daemon/daemon.go

```diff
@@ -973,6 +974,20 @@ func (daemon *Daemon) setHostConfig(container *Container, hostConfig *runconfig.
 
 	container.Lock()
 	defer container.Unlock()
+
+	var stat syscall.Stat_t
```

Comment:
@calavera, yes, it seems windows doesn't support syscall.  Then, I don't know how to deal with it on windows. could give me any ideas?